assets/scss/modules/_tables.scss

```
@@ -39,21 +39,55 @@
 
 .dp-library {
   @mixin dp-c-table($bg-color-zebra, $font-size-table, $bg-color-hover) {
+    .dp-expand-results {
+      position: relative;
+      padding: $dp-spaces--lv1 $dp-spaces--lv3 $dp-spaces--lv0 $dp-spaces--lv0;
+
+      .icon-arrow-next {
+        position: absolute;
+        transform: rotate(0deg);
+      }
+
+      &.dp-rgpd-open-results {
```

Shouldn´t this be changed also?
```suggestion
      &.dp-expand-open-results {
```